autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -264,6 +285,10 @@ def fit(train_data, label, tuning_data=None, time_limits=None, output_directory=
             Higher levels correspond to more detailed print statements (you can set verbosity = 0 to suppress warnings).
             If using logging, you can alternatively control amount of information printed via `logger.setLevel(L)`,
             where `L` ranges from 0 to 50 (Note: higher values of `L` correspond to fewer print statements, opposite of verbosity levels)
+        epochs : int
+            Maximum number of epochs (max_t of HyperbandScheduler)
+        scheduler_options : dict
+            Extra arguments passed to __init__ of scheduler
```

Comment:
I think documentation of `scheduler_options` could briefly explain what scheduler even is, as many users may be unfamiliar with it, eg: 

"Extra arguments passed to __init__ of scheduler to configure the orchestration of training jobs during hyperparameter-tuning.  This is ignored if `hyperparameter_tune=False`.

Reply:
OK